lib/suspenders/generators/testing_generator.rb

```diff
@@ -3,7 +3,7 @@
 module Suspenders
   class TestingGenerator < Generators::Base
     def add_testing_gems
-      gem "rspec-rails", "~> 3.6", group: %i[development test]
+      gem "rspec-rails", "~> 5.1", group: %i[development test]
```

Comment:
this might not have been necessary but it seems like a good idea to upgrade in any case.